f34-py39/Dockerfile

```
@@ -26,13 +26,13 @@ LABEL summary="$SUMMARY" \
 
 USER 0
 COPY ./s2i_assemble.patch /tmp/s2i_assemble.patch
+COPY ./requirements.txt /opt/app-root/src/requirements.txt
```

There is an env variable for the path to the source code you might want to use.

```suggestion
COPY ./requirements.txt $HOME/requirements.txt
```

---

Thanks!! Adjusted in https://github.com/thoth-station/s2i-thoth/pull/210/commits/0e4cc839938df4bcae82aa79598657bd6162596e